example/requirements.txt

```diff
@@ -1,3 +0,0 @@
-mkdocs
-mkdocs-print-site-plugin
-mkdocs-material
```

Comment:
Please revert the deletion of this file. mkdocs-print-site-plugin, mkdocs-material and mkdocs-drawio are independent. Users need to decide which dependencies they want to install

Reply:
`requirements.txt` is an old method and poetry group dev is made for this. 

```bash
poetry install # With dev deps.
poetry run mkdocs serve -f example/mkdocs.yml
```

We can keep the site for testing. We can generate (build) the site to ensure everything works as expected. Though, since we rely on javascript to render the figures into SVG it could be a bit tricky to test. I might come with a solution.  What do you think?